TVOSExample/android/gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.12-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.14.3-bin.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.14.3-all.zip
```

We use all. It also fetches source code & docs then, which is sometimes useful in debugging. I know this is a tvOS application, but just to make it aligned with other apps.